frontend/src/job/JobDetails.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+    <div>Todo: Show status of given job</div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+@Component
+export default class JobDetails extends Vue {}
```

Comment:
Die Component wird noch nicht genutzt, oder?

In der jetzigen Phase ist das natürlich eigentlich kein Problem und es ist ja auch klar, dass das nur ein Zwischenstand ist. Ich würde aber gerne von Anfang an darauf achten, dass kein ungenutzter bzw. unfertiger Code im Repo landet. Auch so im Sinne von continuous deployment fände ich es gut, wenn wir darauf achten würden, dass jeder commit tendenziell funktionsfähig ist.

Reply:
Zu der Component wird weitergeleitet, wenn man einen Job in der Liste anklickt. Ich hatte es jetzt mit reingenommen um ggf. beim Review nochmal zu diskutieren wie wir die einzelnen Jobs jetzt intern abbilden und wie die Routes aussehen sollen.

Reply:
Ah, das hatte ich übersehen. Hatte es so verstanden, dass das die component für die Anzeige der der Job-Infos innerhalb der Liste werden soll, aber die ist ja im Router verlinkt. Ich nehme alles zurück ... :)